### PR TITLE
stored xss fix

### PR DESCRIPTION
**Pull request by mattkrick**
usually i'd quietly merge security things, but this is such an edge case I don't mind making it public.
this also gets us 1 step closer to removing redux-form, which will unlock the ability to migrate to the new react

TEST

- [ ] can upload new user profile pic
- [ ] can upload new org profile pic
- [ ] if i upload a svg with a script tag in it, i see an easter egg

to reproduce the last step:
- open up any of our svg images (eg mark-white.svg)
-  on the 2nd to last line add a tag like:
```
<script>
        console.log('HACKED');
    </script>
```
- save the file & then try to upload it

### CODE REVIEW

File: src/universal/components/AvatarInput.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    if (this.inputRef.current) {
+      this.inputRef.current.click()
+    }
+  }
+  onChange = (e: React.ChangeEvent<HTMLInputElement> | React.FormEvent<HTMLFormElement>) => {
+    const {onSubmit} = this.props
+    const {files} = e.currentTarget
+    const imageToUpload = files ? files[0] : null
+    if (!imageToUpload) return
+    // TODO see if we still need the setTimeout
```
Comment (jordanh) [inline src/universal/components/AvatarInput.tsx]: I dare to ask, what happened without the setTimeout()?

Comment (mattkrick) [reply src/universal/components/AvatarInput.tsx]: actually that's a great question. redux-form needed that, but now that we're vanilla, i think we're OK

**Comment by jordanh** [comment]
# Test Cycle 2

- [x] can upload new user profile pic
- [x] can upload new org profile pic
- [x] if i upload a svg with a script tag in it, i see an easter egg

## Notes

Huzzah! Verified fixes.

*TS Warnings:*

```
WARNING in [at-loader] ./src/universal/components/OrgAvatarInput.tsx:1:25
    TS2306: File '/Users/jordanhusney/Source/github/Parabol/action.git/node_modules/@mattkrick/sanitize-svg/dist/types/sanitizeSVG.d.ts' is not a module.

WARNING in [at-loader] ./src/universal/components/UserAvatarInput.tsx:1:25
    TS2306: File '/Users/jordanhusney/Source/github/Parabol/action.git/node_modules/@mattkrick/sanitize-svg/dist/types/sanitizeSVG.d.ts' is not a module.
```
